spec/timezone.html

```diff
@@ -5,6 +5,65 @@
+
+      <p>
+        An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the *IsValidTimeZoneName* abstract operation as specified in the ECMA-402 specification.
+        If an ECMAScript implementation does not include the ECMA-402 API the following specification of the *IsValidTimeZoneName* abstract operation is used.
+      </p>
+      <p>
+        The IsValidTimeZoneName abstract operation verifies that the _timeZone_ argument (which must be a String value) represents a valid built-in time zone.
+      </p>
+      <p>
+        The abstract operation returns true if _timeZone_, converted to upper case as described in <emu-xref href="#sec-string.prototype.touppercase"></emu-xref>, is equal to one of the built-in time zone names, converted to upper case as described in <emu-xref href="#sec-string.prototype.touppercase"></emu-xref>. It returns false otherwise.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-canonicalizetimezonename" aoid="CanonicalizeTimeZoneName">
+      <h1>CanonicalizeTimeZoneName ( _timeZone_ )</h1>
+
+      <p>
+        An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the *CanonicalizeTimeZoneName* abstract operation as specified in the ECMA-402 specification.
+        If an ECMAScript implementation does not include the ECMA-402 API the following specification of the *CanonicalizeTimeZoneName* abstract operation is used.
+      </p>
+      <p>
+        The CanonicalizeTimeZoneName abstract operation returns the canonical and case-regularized form of the _timeZone_ argument (which must be a String value that is a valid time zone name as verified by the IsValidTimeZoneName abstract operation). The following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _builtinTimeZone_ be the built-in time zone name such that _timeZone_, converted to upper case as described in <emu-xref href="#sec-string.prototype.touppercase"></emu-xref>, is equal to _builtinTimeZone_, converted to upper case as described in <emu-xref href="#sec-string.prototype.touppercase"></emu-xref>.
+        1. Return _builtinTimeZone_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-defaulttimezone" aoid="DefaultTimeZone">
+      <h1>DefaultTimeZone ( )</h1>
+
+      <p>
+        An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the *DefaultTimeZone* abstract operation as specified in the ECMA-402 specification.
+        If an ECMAScript implementation does not include the ECMA-402 API the following specification of the *DefaultTimeZone* abstract operation is used.
+      </p>
+      <p>
+        The DefaultTimeZone abstract operation returns a String value representing the valid (<emu-xref href="#sec-isvalidtimezonename"></emu-xref>) and canonicalized (<emu-xref href="#sec-canonicalizetimezonename"></emu-xref>) time zone name for the host environment's current time zone.
```

Comment:
perhaps this could be a bit more explicit - with Asserts that `IsValidTimeZoneName()` is true?

Reply:
^ agreed